packages/kit/test/apps/dev-only/package.json

```
@@ -1,5 +1,5 @@
 {
-	"name": "test-basics",
+	"name": "test-no",
```

this is called `test-dev-only` in `master`, which is a better name
```suggestion
	"name": "test-dev-only",
```